resources/sts/4.13/openshift_aws_vpce_operator_avo_aws_creds_policy.json

```diff
@@ -17,12 +17,40 @@
         "ec2:CreateVpcEndpoint",
         "ec2:DeleteVpcEndpoints",
         "ec2:DescribeVpcEndpoints",
+        "ec2:DescribeVpcs",
         "ec2:ModifyVpcEndpoint",
+        "ec2:DescribeVpcEndpointServices",
+        "route53:ChangeResourceRecordSets",
```

Comment:
Seems like there is duplication here with both the line below and the other section in Route53ManageRecords